frontend/puppeteerTest.js

```diff
@@ -560,6 +565,54 @@ const resumeSession = async(page) => {
     await passed(13);
 };
 
+const renameAndRevertColumnName = async(page) => {
+    await page.waitForSelector(homePage.customerTable);
```

Comment:
I had a hard time following what is happening in the test. Can you please add block comments. I see that the rest of the file doesn't have block comments and that's something we missed in previous reviews.

Basically some commentary that guides reader through the test. Something like:

// Comment 1
Some block of code

// Comment 2
Next block of code

and so on